docs/tutorials/Remark.adoc

```diff
@@ -0,0 +1,396 @@
+ * @return           ArgumentMultimap object that maps prefixes to their
+ * arguments
+ */
+----
+
+We can tell `ArgumentTokenizer#tokenize()` to look out for our new prefix `r/` and it will return us an instance of `ArgumentMultimap`.
+Now let's find out what we need to do in order to obtain the Index and String that we need.
+Let's look through `ArgumentMultimap` :
+
+[source, java]
+.ArgumentMultimap.java
+----
+/**
+ * Returns the last value of {@code prefix}.
+ */
+public Optional<String> getValue(Prefix prefix) {
+    List<String> values = getAllValues(prefix);
+    return values.isEmpty() ? Optional.empty() :
+        Optional.of(values.get(values.size() - 1));
+}
+----
+
+This appears to be what we need to get a String of the remark.
+But what about the Index? Taking a quick peek at existing an `Command`...
+
+[source, java]
+.DeleteCommandParser.java
+----
+Index index = ParserUtil.parseIndex(args);
+return new DeleteCommand(index);
+----
+
+There appears to be another utility class that obtains an `Index` from the input provided by the user.
+
+Now that we have the know-how to extract the data that we need from the user's input, we can create a new instance of `RemarkCommand`.
+
+[source, java]
+.RemarkCommandParser.java
+----
+public RemarkCommand parse(String args) throws ParseException {
+    requireNonNull(args);
+    ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args,
+        PREFIX_REMARK);
+
+    Index index;
+    try {
+        index = ParserUtil.parseIndex(argMultimap.getPreamble());
+    } catch (IllegalValueException ive) {
+        throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+            RemarkCommand.MESSAGE_USAGE), ive);
+    }
+
+    String remark = argMultimap.getValue(PREFIX_REMARK).orElse("");
+
+    return new RemarkCommand(index, remark);
+}
+----
+
+NOTE: Don't forget to update `AddressBookParser` to use our new `RemarkCommandParser`!
+
+If you are stuck, check out the sample link:https://github.com/j-lum/addressbook-level35/commit/c0f8419be4fb59bffafcd73026769459e5cf5c5e#diff-fc19ecee89c3732a62fbc8c840250508[here].
+
+== Add `Remark` to the model
+
+Now that we have all the information that we need, let's lay the groundwork for some _persistent_ changes.
+We achieve that by working with the `Person` model.
+Each field in a Person is implemented as a separate class (e.g. a `Name` object represents the person's name).
+That means we should add a `Remark` class so that we can use a `Remark` object to represent a remark given to a person.
+
+=== Add a new `Remark` class
+
+Create a new `Remark` in `seedu.address.model.person`. Since a `Remark` is a field that is similar to `Address`, we can reuse a significant bit of code.
+
+A quick copy-paste and search-replace later, you should have something like link:https://github.com/j-lum/addressbook-level35/commit/c8e6048f0d81f2b4b7797e838226da823892129b#diff-af2f075d24dfcd333876f0fbce321f25[this].
+
```

Comment:
TL;DR Overall, this part is fine.

This is a relatively hard part for lazy students who are simply zombie-ing and copy pasting to go through the tutorial. To make a remark class in the model requires someone to recognize that there are no constraints to remarks, hence no need to `checkArguments()` and check an Remark equivalent for `isValidAddress`

I would argue it's some copy and paste, with some required thinking, not necessarily a 'quick' one

Anyways, someone's who's lazy can just check the Github repo and copy and whole Remark.java wholesale, so it's fine overall

Reply:
I have a very nasty feeling some students are going to clone the last step of the tutorial and call it a day. I was thinking of inserting some gotchas to distinguish between students who actually attempted  the tutorial vs just cloning but I decided against it since they are responsible for their own learning I guess. 

I'll pull the 'quick' out and add a tip to make them consider what they need to copy and paste in.